Aztec/Classes/NSAttributedString/HTML/HTMLRepresentation.swift

```diff
@@ -2,13 +2,67 @@ import Foundation
 
 /// This enum specifies the different entities that can represent a style in HTML.
 ///
-enum HTMLRepresentation {
-    case attribute(Attribute)
-    case element(HTMLElementRepresentation)
-    case inlineCss(CSSProperty)
+class HTMLRepresentation: NSObject {
+    enum Kind {
+        case attribute(Attribute)
+        case element(HTMLElementRepresentation)
```

Comment:
How about renaming HTMLElementRepresentation to HTMLElement or Element to match the other cases?